{{ cookiecutter.project_slug }}/pyproject.toml

```
@@ -121,6 +121,15 @@ convention = "numpy"
 [tool.ruff.lint.pep8-naming]
 ignore-names = ["df", "fn", "ca", "tp", "x", "i"]
 
+[tool.ruff.lint.flake8-import-conventions.extend-aliases]
+# Recommended aliases for caf packages
+"caf.toolkit" = "ctk"
+"caf.base" = "cb"
+"caf.mat" = "cm"
+"caf.space" = "cs"
+"caf.distribute" = "cdb"
+"caf.viz" = "cvz"
```

Some of these might be too short and edging on becoming cryptic in the middle of code. I think the `caf.` is the main issue, so to keep this clear and consistent how about:


```suggestion
"caf.toolkit" = "ctk"
"caf.base" = "cbase"
"caf.mat" = "cmat"
"caf.space" = "cspace"
"caf.distribute" = "cdist"
"caf.viz" = "cviz"
```